### PR TITLE
Billing and subscription FAQ updates

### PR DESCRIPTION
**Pull request by stephaurelio**
### Proposed changes

This PR edits the FAQs for billing and subscription and moves relevant content to the corresponding manuals pages. It also includes a diagram to clarify subscriptions (see [Deploy Preview](https://deploy-preview-19182--docsdocker.netlify.app/subscription/details/) to view diagram).

### Related issues (optional)

Closes JIRA https://docker.atlassian.net/browse/ENGDOCS-1939 & https://docker.atlassian.net/browse/ENGDOCS-1862

### CODE REVIEW

File: content/subscription/downgrade.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ When you downgrade your subscription, access to paid features is available until
 
 Before you downgrade to a free subscription, you may need to reduce the number of team members and convert any private repositories to public repositories or delete them. For information, see [Docker Pricing](https://www.docker.com/pricing).
 
+> **Important**
+>
+> If you downgrade from a Pro subscription to a Personal subscription, note that Personal subscriptions don't include collaborators for private repositories. If you have collaborators when you move to a Personal subscription from a paid subscription you need to remove all collaborators. If you haven't removed collaborators within 30 days of moving to a free plan, all collaborators are automatically removed from your account.
```
Comment (ob1dev) [inline content/subscription/downgrade.md:18]: **suggestion:** I would be more specific here.

```suggestion
> If you downgrade your personal account from a Pro subscription to a Personal subscription, note that Personal subscriptions don't include collaborators for private repositories. If you have collaborators when you move to a Personal subscription from a paid subscription you need to remove all collaborators. If you haven't removed collaborators within 30 days of moving to a free plan, all collaborators are automatically removed from your account.
```

File: content/subscription/details.md [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
 
-You can do more with Docker with a Docker subscription, such as add collaborators, create scoped access tokens, and create private repositories. This page provides an overview of each subscription tier. To compare features available in each tier, see [Docker Pricing](https://www.docker.com/pricing/).
+You can do more with Docker with a Docker Core subscription, such as add collaborators, create scoped access tokens, and create private repositories. This page provides an overview of each subscription tier. To compare features available in each tier, see [Docker Pricing](https://www.docker.com/pricing/).
 
-You can also subscribe to Docker Scout. See [Docker Scout subscriptions and features](./scout-details.md).
+The following describes some of the key features included with your Docker Core subscription:
+
+![Docker Core subscription diagram](images/subscription-diagram.webp)
+
+Docker Scout is included for 3 repositories. You can purchase a Docker Scout subscription to enable more repositories. See [Docker Scout subscriptions and features](./scout-details.md).
```
Comment (ob1dev) [inline content/subscription/details.md:13]: **suggestion:** I belive Scout team started using term `Docker Scout-enabled repositories`.

```suggestion
3 Docker Scout-enabled repositories for advanced remote image analysis. You can purchase a Docker Scout subscription to enable more repositories. See [Docker Scout subscriptions and features](./scout-details.md).
```

**Review by ob1dev — APPROVED** [review]
LGTM, but left some suggestions.

File: content/subscription/downgrade.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (stephaurelio) [reply content/subscription/downgrade.md:18]: Good question this was in the original FAQ so I'll have to look in to the original source. Will verify if this number is correct.

[the remark this answers is not in the document]
Comment (stephaurelio) [reply content/subscription/downgrade.md]: Confirmed with Jeff Carter what you wrote above is accurate. Updating this callout!

**Review by dvdksn — APPROVED** [review]
LGTM

**Pull request merged**